packages/admin-panel/src/routes/surveys/optionSets.js

```diff
@@ -12,13 +12,22 @@ const OPTION_SET_FIELDS = [
 
 const OPTION_SET_COLUMNS = [
   ...OPTION_SET_FIELDS,
+  {
+    Header: 'Export',
+    // source: 'id',
+    type: 'export',
+    actionConfig: {
+      exportEndpoint: 'optionSet',
+      fileName: '{name}',
+    },
+  },
```

Comment:
I based this off the export column from the survey responses page/tab; but I’m not actually sure what some of these properties do. In particular, the `source` property

Option set export seems to be working well without it, but I’m not sure if it _should_ have a `source` set. Keeping it as `'id'` causes a crash because the Edit column also uses ` source: 'id'`